### PR TITLE
added all CRUD for posts

### PR DESCRIPTION
**Pull request by d0m1ngo**

### CODE REVIEW

File: database/data.json [hunk 1]
```diff
@@ -1,10 +1 @@
...
-  "posts": [
-    {
-      "id": 1,
-      "text": "Some text post 1",
-      "title": "Some title post 1"
-    }
-  ],
-  "comments": [{ "id": 1, "title": "Comment1", "text": "Comment1", "createdDate": 0, "postId": 1 }]
-}
+{"posts":[{"id":1,"text":"WAZZAp","title":"Some title post 1"},{"text":"some text2","title":"some title2","id":"03463cf5-372a-43c2-a58b-9b40dd1185cb"},{"text":"some text2444","title":"some title212124","id":"ee5c5d9a-1fcc-435f-be29-5b4aeff3e213"}],"comments":[{"id":1,"title":"Comment1","text":"Comment1","createdDate":0,"postId":1}]}
```
Comment (demns) [inline database/data.json]: try third parameter in JSON.stringify (or whatever else) to beautify it for yourself

File: package.json [hunk 2]
```diff
@@ -12,15 +12,15 @@
     "express-session": "^1.15.6",
     "swagger-ui": "^3.23.0",
     "swagger-ui-express": "^4.0.7",
+    "uuid": "^3.3.2",
```
Comment (demns) [inline package.json]: any package-lock changes?

File: package.json [hunk 3]
```diff
@@ -12,15 +12,15 @@
...
     "winston": "^3.2.1",
-    "yamljs": "^0.3.0",
-    "nodemon": "1.19.1"
+    "yamljs": "^0.3.0"
   },
   "devDependencies": {
     "chai": "^4.2.0",
```
Comment (demns) [inline package.json]: please remind me of having conversation with you about devDependencies

**Comment by demns** [comment]
ок короч)

File: package.json (hunk 2 above)
Comment (d0m1ngo) [reply package.json]: yarn-lock changed

**Pull request merged**